core/src/main/java/org/infinispan/util/concurrent/locks/impl/DefaultPendingLockManager.java

```diff
@@ -343,29 +411,30 @@ public long getRemainingTimeout() {
       }
 
       @Override
-      public void onCompletion() {
+      public Void call() throws Exception {
+         isReady();
+         return null;
+      }
+
+      private void onRelease() {
          if (isReady()) {
             notifier.complete(null);
          }
       }
 
       @Override
-      public Void call() throws Exception {
-         isReady();
-         return null;
-      }
-
-      private PendingTransaction getTimedOutTransaction() {
+      public PendingTransaction getTimedOutTransaction() {
```

Comment:
I think the name is misleading, because the transaction you're returning hasn't timed out... why not call it just `pendingTransaction`?